src/static/css/Partners.css

```diff
@@ -63,19 +63,23 @@
 
 .kia {
   position: absolute;
-  width: 182px;
-  height: 182px;
+  /* width: 182px;
+  height: 170px; */
+  width: 12%;
+  height: 16%;
```

Comment:
In this case it's you want the logos to maintain the same aspect ratio, so maybe set both width and height in terms of vw/vh for all logos.